src/components/AddImage.jsx

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+function AddImage() {
+  let [link, setLink] = useState('');
+  let [files, setFiles] = useState(null);
+
+  function handleFileChange(e) {
+    const { files } = e.target;
+    setFiles(files[0]);
+
+    if (files.length === 0) {
+      setLink('');
+    } else {
+      const imageURL = window.URL.createObjectURL(files[0]);
+      setLink(imageURL);
+    }
+  }
+
+  async function submitImg() {
+    console.log('files', files, 'link', link);
+    let form = new FormData();
+    form.append('image', files);
+    form.append('album', 'PZijGmD');
+
+    await axios({
+      method: 'POST',
+      url: 'https://api.imgur.com/3/image',
+      headers: {
+        Authorization: `Bearer access token`,
+      },
+      data: form,
+      mimeType: 'multipart/form-data',
+    })
+      .then(res => {
+        console.log(res);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }
```

Comment:
`async/await` and `Promise.then().catch()` are do the same thing, mix them is not a good approach.
if use `async/await`, here can use `try/catch` to instead `.then().catch()`